src/App.js

```diff
@@ -1,24 +1,18 @@
-import logo from './logo.svg';
+import { Route, Switch } from 'react-router-dom';
 import './App.css';
+import Header from './components/Header/Header';
+import RGB from './components/RBG/RGB';
+import NotFound from './components/NotFound/NotFound';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <main className="App">
+      <Header />
+      <Switch>
+        <Route path="/RGB/:R/:G/:B" component={RGB} />
```

Comment:
This is working, but there is no path for when the page loads. For example, "/" will always be the first path that loads; without giving that path a component or redirecting, your app always starts with a "404 not found" in its current state. If you would like some advice on handling this, let me know!